pages/national_resources/no_resources.md

```diff
@@ -297,9 +299,9 @@ Established by ELIXIR Norway and hosted by the University of Oslo, the Norwegian
 
 ELIXIR Norway's tools assembly for personally identifiable datasets is based on the University of Oslo's Services for Sensitive Data (TSD). This infrastructure provides resources to help you comply with Norwegian and European regulations regarding sensitive personal data. You can also store, process, and analyse your data in a secure, restricted environment and use a wide range of integrated data management tools. More details are available on the [National Norwegian services for sensitive (personal) data tool assembly RDMkit page](tsd_assembly).
```

Comment:
Replace "This infrastructure" with "The TSD" or "The TSD infrastructre"

Reply:
```suggestion
ELIXIR Norway's tools assembly for personally identifiable datasets is based on the University of Oslo's Services for Sensitive Data (TSD). TSD provides resources to help you comply with Norwegian and European regulations regarding sensitive personal data. You can also store, process, and analyse your data in a secure, restricted environment and use a wide range of integrated data management tools. More details are available on the [National Norwegian services for sensitive (personal) data tool assembly RDMkit page](tsd_assembly).
```